src/app/modules/customer-management/components/customer-info/components/create-customer/create-customer.ts

```diff
@@ -3,8 +3,14 @@ import { FormGroup, FormBuilder, Validators } from '@angular/forms';
 import { Store, select } from '@ngrx/store';
 
 import { Subscription } from 'rxjs';
-import { CustomerState, CreateCustomer, LoadCustomers } from 'src/app/modules/customer-management/store';
-import { getStatusMessage } from './../../../../store/customer-management.selectors';
+import { getStatusMessage, getCustomerById } from './../../../../store/customer-management.selectors';
+import { Customer } from 'src/app/modules/shared/models/index';
```

Comment:
import should be
import { Customer } from 'src/app/modules/shared/models';

Reply:
done